### PR TITLE
build: Remove upper bound constraint from base

### PR DESCRIPTION
**Pull request by mark-stopka**
- Enable GHC 9.0.x build

### CODE REVIEW

**Comment by mark-stopka** [comment]
See attached GHC 9.0.0 build log file...
[ghc-9.0.0-ghc-ekg-json.build.log](https://github.com/tibbe/ekg-json/files/5681674/ghc-9.0.0-ghc-ekg-json.build.log)

File: ekg-json.cabal [hunk 1]
```diff
@@ -25,7 +25,7 @@ library
     System.Metrics.Json
   build-depends:
     aeson >=0.4 && < 1.6,
-    base >= 4.6 && < 4.15,
+    base >= 4.6,
```
Comment (23Skidoo) [inline ekg-json.cabal]:
```suggestion
    base >= 4.6 && < 4.16,
```

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Comment by mark-stopka** [comment]
Thanks! Could you please also bump it up on Hackage? At [openSUSE we have automated Hackage2RPM workflow](https://github.com/opensuse-haskell/configuration), so I could then remove this specific patch....

**Comment by 23Skidoo** [comment]
I'll bump the bounds on Hackage once GHC 9.0.1 final is out.

**Comment by mark-stopka** [comment]
Ok, thx.

**Pull request merged**